solvers/lasso_jl.py

```diff
@@ -0,0 +1,80 @@
+from pathlib import Path
+
+import numpy as np
+from benchopt import safe_import_context
+from benchopt.helpers.julia import (JuliaSolver, assert_julia_installed,
+                                    get_jl_interpreter)
+from benchopt.runner import INFINITY
+from benchopt.stopping_criterion import SufficientProgressCriterion
+
+with safe_import_context() as import_ctx:
+    assert_julia_installed()
+    from scipy import sparse
+
+
+# File containing the function to be called from julia
+JULIA_SOLVER_FILE = str(Path(__file__).with_suffix(".jl"))
+
+
+class Solver(JuliaSolver):
+    name = "lasso_jl"
+    stopping_criterion = SufficientProgressCriterion(
+        patience=7, eps=1e-15, strategy="tolerance"
+    )
+    julia_requirements = [
+        "Distributions",
+        "GLM",
+        "Lasso",
+        "LinearAlgebra",
+        "PyCall",
+        "SparseArrays",
+    ]
```

Comment:
I would remove the ones that are deps from `Lasso` to leave it to the `Lasso.jl` package to manage its deps.

```suggestion
    julia_requirements = [
        "Lasso",
        "PyCall",
        "SparseArrays",
    ]
```